README.md

```diff
@@ -1 +1,61 @@
-# SENG 499 - Company 2 - Algorithm 1 
\ No newline at end of file
+# SENG 499 - Company 2 - Algorithm 1 
+
+This repository contains a package, [Course-Scheduler](https://pypi.org/project/coursescheduler/), for
+that returns a course schedule for three terms of Software Engineering courses at UVic. The Course Scheduler uses the 
+Constraint Satisfaction Problem (CSP) Algorithm to determine the hard constraint of the schedule, and produces a schedule 
+that adheres to the constraints. The scheduler also takes into account soft constraints for professor preferences, 
+however, these constraints are not always fulfilled. 
+
+This package was created for the Capstone SENG499 class: Company 2 Course Scheduler Project. 
+Authored by the Company2-Algorithm1 sub-team.
+
+## Install from PyPi Using PIP
+Install this package into your environment from PyPi using `pip`. 
+
+```bash
+$ pip install coursescheduler
+```
+The most recent available version of the package is uploaded to the test PyPi index automatically.
+To ensure that you are working with the most recent release upgrade this module before integrating.
+
+```bash
+$ python3 -m pip install --upgrade coursescheduler
+```
+
+## Install from Local Archives
+
+Clone this repo into your repository. Inside the algorithm 1 module directory,
+build and install the package as shown below. 
+
+```bash
+$ python3 -m build
+$ pip3 install coursescheduler-0.0.1.tar.gz
+```
+
+## Usage
+The algorithm 1 module may then be imported into and called from the backend subteam. In the example below
+`historical_course_data` `professors` and `schedule` are python dictionaries or
+JSON strings. A schedule object with capacities assigned is encoded as a JSON string and returned
+to caller.
```

Comment:
This describes algorithm 2. Also our inputs cannot be JSON strings and our output is not encoded as a JSON string. Suggested fix:
```suggestion
Once installed, the algorithm can be imported and called with `generate_schedule`, as shown below. `generate_schedule` expects three parameters and returns a single output, all of which are Python dictionaries. The API specification can be found [here](https://docs.google.com/document/d/163L7pv6w5Z38rUrl2EwRJq-A9ZLllCIO9uYbUkdxi2s/edit?usp=sharing).
```